erpnext/selling/report/sales_report_by_customer/sales_report_by_customer.py

```diff
@@ -30,13 +30,7 @@ def run(self):
 		return self.columns, self.data, None, self.chart
 
 	def get_columns(self):
-		self.columns = [{
-				"label": _(self.filters.tree_type + " ID"),
-				"options": self.filters.tree_type if self.filters.tree_type != "Order Type" else "",
-				"fieldname": "entity",
-				"fieldtype": "Link" if self.filters.tree_type != "Order Type" else "Data",
-				"width": 140 if self.filters.tree_type != "Order Type" else 200
-			}]
+		self.columns = []
 
 		if self.filters.tree_type in ["Customer", "Supplier", "Item"]:
 			self.columns.append({
```

Comment:
- If this is Sales Report by Customer, there should a link field for customer in here

Reply:
Where, it is already there I think!

Reply:
It's actually consist of Item and Customer both that's why it's not specific to link to customer!
and the name was mentioned as Sales Report by Customer, that's why i changed it to Sales Report By Customer